docs/modules/ROOT/pages/extensions.adoc

```diff
@@ -250,7 +250,7 @@ Please see the documents below for more formatter API details.
 
 * https://www.rubydoc.info/gems/rubocop/RuboCop/Formatter/BaseFormatter[RuboCop::Formatter::BaseFormatter]
 * https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Offense[RuboCop::Cop::Offense]
-* https://www.rubydoc.info/gems/parser/Parser/Source/Range[Parser::Source::Range]
+* https://whitequark.github.io/parser/Parser/Source/Range.html[Parser::Source::Range]
```

Comment:
This is not really ideal, that page is pretty much dead. It was last updated 10 years ago (https://github.com/whitequark/parser/issues/806). Looks like the link format changed, so it should now be https://www.rubydoc.info/github/whitequark/parser/Parser/Source/Range.

Although, that website was always horribly slow for me and it actually gives me a cloudflare error right now so I wouldn't mind a different link.

Reply:
Hm, Rubocop itself is still available at the old url so I guess it's a bug with rubydoc itself that it doesn't work now.

Reply:
Exists where? I think it was already mentioned that the self-hosted Parser docs haven't been updated in 10 years.

Reply:
Oh, I misunderstood.

Reply:
I guess it'd be best to use gemdocs.org instead.